### PR TITLE
My resume builder submission

### PR DESCRIPTION
**Pull request by code-nv**
Wasn't able to get through the full exercise, got caught up trying to allow the user to input additional work experience.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>Resume Builder</title>
+	</head>
+	<body>
+		<form action="#">
```
Comment (imelendez) [inline index.html:17]: I used form ID so later I can have javascript do work on the form

I read there might be a way to transform form to JSON // it was a jquery function

File: index.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+			<fieldset class="basics" data-jsonSection="basics">
+				<label for="name">full name</label>
+				<input type="text" id="name" />
+				<label for="label">current/desired position</label>
+				<input type="text" id="label" />
+				<label for="picture">profile Picture</label>
+				<input type="image" id="picture" />
+				<label for="email">email</label>
```
Comment (imelendez) [inline index.html]: I also noted the schema for this was long. So first tried to attempt my goals/functions on a simple form. Then I could scale to this larger schema. 

Tip for future when taking on a big (relative) task like this. Start small.

**Review by imelendez — COMMENTED** [review]
Good job! Left some general tips.

File: script.js [hunk 3]
```diff
@@ -98,3 +99,57 @@ Resume = {
...
+	$(`form`).on("submit", function(e) {
+		e.preventDefault();
+		app.updateJSON();
+	});
+
+	$(".moreWork").on("click", function() {
+		app.moreWork();
+	});
+};
+
```
Comment (Aliceeeee825) [inline script.js:152]: Good job using namespace

File: script.js [hunk 4]
```diff
@@ -98,3 +99,57 @@ Resume = {
...
 };
+app.workCounter = 0;
+
+app.moreWork = function() {
+	// append more work info
+	app.workCounter++;
+	console.log("fizz");
+	const workToAppend = `<fieldset class="work${app.workCounter}" data-jsonSection="work${app.workCounter}">${$(
+		".work0"
+	).html()}</fieldset>`;
```
Comment (Aliceeeee825) [inline script.js:110]: I love the way you give each one a unique number

File: style.css [hunk 5]
```diff
@@ -0,0 +1,30 @@
...-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}button,html [type=button],[type=reset],[type=submit]{-webkit-appearance:button}button::-moz-focus-inner,[type=button]::-moz-focus-inner,[type=reset]::-moz-focus-inner,[type=submit]::-moz-focus-inner{border-style:none;padding:0}button:-moz-focusring,[type=button]:-moz-focusring,[type=reset]:-moz-focusring,[type=submit]:-moz-focusring{outline:1px dotted ButtonText}fieldset{padding:.35em .75em .625em}legend{box-sizing:border-box;color:inherit;display:table;max-width:100%;padding:0;white-space:normal}progress{display:inline-block;vertical-align:baseline}textarea{overflow:auto}[type=checkbox],[type=radio]{box-sizing:border-box;padding:0}[type=number]::-webkit-inner-spin-button,[type=number]::-webkit-outer-spin-button{height:auto}[type=search]{-webkit-appearance:textfield;outline-offset:-2px}[type=search]::-webkit-search-cancel-button,[type=search]::-webkit-search-decoration{-webkit-appearance:none}::-webkit-file-upload-button{-webkit-appearance:button;font:inherit}details,menu{display:block}summary{display:list-item}canvas{display:inline-block}template{display:none}[hidden]{display:none}
```
Comment (monarchwadia) [inline style.css:1]: Usually, leaving condensed (also sometimes called "uglified")  CSS like this causes heartache in the long run, when you want to change CSS values quickly, or debug a bit of styling.

Maybe run a prettifier on it?

File: index.html (hunk 2 above)
Comment (code-nv) [reply index.html]: I got way too caught up in this markup I wish I had spent more time on the js side of things considering this was a js assignment

File: script.js (hunk 4 above)
Comment (code-nv) [reply script.js:110]: thank you! I got kind of caught up in this hahaha

File: style.css (hunk 5 above)
Comment (code-nv) [reply style.css:1]: this is just my default setup snippet! I never touch this and usually I can tuck it away in a sass partial but I felt the scope of the project was too small to bother with it. I didn't spend much time in my css or run my prettier formatting on it at all, thank you for the heads up for my future head aches though!

**Pull request closed without merging**